### PR TITLE
Implementation of mov op

### PR DESCRIPTION
**Pull request by luizperes**
This PR implements what is described on #67

### CODE REVIEW

File: tests/mov.brain [hunk 1]
```diff
@@ -0,0 +1 @@
++++++>++*&^#
```
Comment (rafaelcn) [inline tests/mov.brain]: I may be blind or something, but tell me that there's 6 `+` instead of 5 @luizperes.

File: tests/mov.cmp0 [hunk 2]
```diff
@@ -0,0 +1,18 @@
+=== Debug Information ===
+IncrementInstr (1)
+IncrementInstr (1)
+IncrementInstr (1)
+IncrementInstr (1)
+IncrementInstr (1)
```
Comment (rafaelcn) [inline tests/mov.cmp0]: Doesn't make sense per previous comment (there's only five `IncrementInstr` and there's 6 `+`)

File: tests/mov.brain (hunk 1 above)
Comment (luizperes) [reply tests/mov.brain]: Hey @rafaelcn, thanks for pointing that! Thank you so much, however, I checked and one of these pluses is from the github 
screen shot 2018-01-31 at 11 59 43 am

It is in the same style and with no spacements, so we seems that it is part of de code xD

Comment (rafaelcn) [reply tests/mov.brain]: Wow, that's awkward. Approved then, gonna merge it.

**Pull request merged**